source/mainnet/smart-contracts/tutorials/fungible-tokens/mint-xfer.rst

```diff
@@ -0,0 +1,147 @@
+.. _mint-xfer-ft:
+
+========================================
+Mint, transfer, and burn fungible tokens
+========================================
+
+Now you are ready to mint your new tokens. Before minting, look at the minting parameters in the JSON file below. You need to specify the owner, then the token data following with its tokenID, metadata location on IPFS with the URL that you got when you uploaded the metadata file, hash value of it, amount to be minted, and maximum supply number. In this case, it uses tokenID 01, and mint 110 initially with a maximum supply of 1000.
+
+.. code-block:: JSON
+
+    {
+        "owner": {
+            "Account": [
+                "3AiAikC2v4H3Rv4L58oHvdX5N8LJoarsQwN3G7oNS7BoFsmt7N"
+            ]
+        },
+        "tokens": [
+            [
+                "01",
+                [
+                    {
+                        "url": "https://gateway.pinata.cloud/ipfs/QmZBrF6HuoN12HyAznyk7gwFpnefooDbfxq3JeKTWToL1W",
+                        "hash": "c00a140ea66af09c343083c0e7ccc77d5e0670d90014569798801bffad637d15"
+                    },
+                    {
+                        "amount": "110",
+                        "max_supply": "1000"
+                    }
+                ]
+            ]
+        ]
+    }
+
+Run the command below to call the contract ``mint()`` function. Remember that this is using the Concordium testnet node; if you are using your own node your command needs to be changed.
```

Comment:
Please add how it needs to be changed (`--grpc-port` and `--grpc-ip` need to be changed (at least the latter, maybe not the former)

Reply:
It shows in the deploy contract part for the rest, I don't think there is a need to point it out again.

Reply:
Ok.